R/module_nested_tabs.R

```diff
@@ -88,9 +88,10 @@ ui_nested_tabs.teal_modules <- function(id, modules, datasets, depth = 0L) {
 #' @export
 #' @keywords internal
 ui_nested_tabs.teal_module <- function(id, modules, datasets, depth = 0L) {
+  ns <- NS(id)
   checkmate::assert_class(datasets, "FilteredData")
   args <- isolate(teal.transform::resolve_delayed(modules$ui_args, datasets))
-  args <- c(list(id = id), args)
+  args <- c(list(id = ns("module")), args)
```

Comment:
line 102 - just checking that that's ok (for example missing_data and variable_browser have data in the UI)